README.md

```diff
@@ -107,6 +107,22 @@ If requests are passed to `QBWC.add_job`, any `QBWC::Worker#requests` method wil
 
 Similarly, a `QBWC::Worker#handle_response` method cannot access variables that are in-memory at the time that `QBWC.add_job` is called; however, you can optionally pass a serializable value (for example, String, Array, or Hash) to `QBWC.add_job`. This data will immediately be persisted by `QBWC.add_job`, then later passed to `QBWC::Worker#handle_response` during a QuickBooks Web Connector session.
 
+### Sessions ###
+
+In certain cases, you may want to perform some initialization prior to each QuickBooks Web Connector session (a QuickBooks Web Connector session is established when you manually run (update) an application's web service in QuickBooks Web Connector, or when QuickBooks Web Connector automatically executes a scheduled update). For this purpose, you may optionally provide an initialization block that will be invoked once when each QuickBooks Web Connector session is established, and prior to executing any queued jobs.
+
+You assign this initialization block prior to any QuickBooks Web Connector session being established, outside of any `QBWC::Worker` classes. For example:
```

Comment:
Not quite following; if you mean a model's initializer, that would invoke extraneous calls to `set_session_initializer` (each time a model object is instantiated); whereas `set_session_initializer` only needs to be called once (analogous to, say, [model validations](http://guides.rubyonrails.org/active_record_validations.html#validations-overview)).

Thinking more, if `set_session_initializer` is invoked from a model initializer, that opens the possibility of dynamically adjusting this _global_ session initializer (good for dynamic operation, but bad in that it could introduce potential for bugs since the most recent invocation of `set_session_initializer` applies globally to all subsequent sessions).